hwilib/cli.py

```diff
@@ -125,7 +128,9 @@ def process_commands(cli_args):
     signmsg_parser.set_defaults(func=signmessage_handler)
 
     getkeypool_parser = subparsers.add_parser('getkeypool', help='Get JSON array of keys that can be imported to Bitcoin Core with importmulti')
-    getkeypool_parser.add_argument('--keypool', action='store_true', help='Indicates that the keys are to be imported to the keypool')
+    kparg_group = getkeypool_parser.add_mutually_exclusive_group()
+    kparg_group.add_argument('--keypool', action='store_const', const=True, help='Indicates that the keys are to be imported to the keypool (default: true)')
+    kparg_group.add_argument('--nokeypool', action='store_const', const=True, help='Indicates that the keys are not to be imported to the keypool (default: false)')
```

Comment:
It would be cleaner to have both of these use `dest='keypool'` and `action='store_true'` (and remove the `const=...`). `--keypool` will need a `default=True` also. Then later we can still use `args.keypool` instead of having to check between both `args.keypool` and `args.nokeypool`.

Also, please remove the `(default: ..)` parts of the help text. We should show the defaults for every option, and argparse has a way to do this without specifying the default in every help time via the use of the `default=...` parameter.

Reply:
done

Reply:
done(though I had to add some more args to make it work, guess and check)